src/components/toggle/toggle_dispatcher.ts

```diff
@@ -0,0 +1,27 @@
+import {Injectable} from '@angular/core';
+
+
+/**
+ * Class for toggles to coordinate unique selection based on name.
+ * Intended to be consumed as an Angular service.
+ * This service is needed because native radio change events are only fired on the item
+ * currently being selected, and we still need to uncheck the previous selection.
+ */
+@Injectable()
+export class MdToggleDispatcher {
```

Comment:
I think we should be able to rename the `MdRadioDispatcher` to something like `MdUniqueSelectionDispatcher`, move it to `core/` and use it in both components.